src/Console/ChromeDriverCommand.php

```diff
@@ -178,21 +148,20 @@ protected function version()
             return $version;
         }
 
-        $version = (int) $version;
+        $milestone = (int) $version;
 
-        if ($version < 70) {
-            return $this->legacyVersions[$version];
-        }
+        $milestones = json_decode($this->getUrl($this->versionsUrl), true);
 
-        return trim($this->getUrl(
-            sprintf($this->versionUrl, $version)
-        ));
+        return $milestones['milestones'][$milestone]['version']
+            ?? throw new Exception('Could not get the ChromeDriver version.');
```

Comment:
I'm throwing these to end the process but not sure if that's the right approach.